packages/ui/components/form-core/src/choice-group/CustomChoiceGroupMixin.js

```diff
@@ -0,0 +1,204 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { ChoiceGroupMixin } from './ChoiceGroupMixin.js';
+
+/**
+ * @typedef {import('../../types/choice-group/CustomChoiceGroupMixinTypes.js').CustomChoiceGroupMixin} CustomChoiceGroupMixin
+ * @typedef {import('../../types/choice-group/CustomChoiceGroupMixinTypes.js').CustomChoiceGroupHost} CustomChoiceGroupHost
+ */
+
+/**
+ * @param {any|any[]} value
+ * @returns {any[]}
+ */
+function normalizeArray(value) {
```

Comment:
Nitpicking: could this be called `ensureArray`?

Reply:
Will do